docs/running-on-yarn.md

```diff
@@ -875,6 +875,7 @@ The following extra configuration options are available when the shuffle service
     initialization. This prevents application failures caused by running containers on
     NodeManagers where the Spark Shuffle Service is not running.
   </td>
+  <td></td>
```

Comment:
Please ensure that the value is fulfilled.

Reply:
+1

Reply:
Okay, let me find the versions of each option.

Reply:
Done